### ISSUE TITLE
Is there any training code?

### ISSUE THREAD

**Issue by dd9991-dd** [DepthAnything/Video-Depth-Anything#51]
Hello, thank you for your contribution to the thesis. 
I have some questions that I would like to ask you. I didn't see the training code part in the code. Could it be that I overlooked it? If it hasn't been released yet, when can it be released? 
Thank you for your reply.

**Comment by guohengkai** [comment]
Hi, we have released our training loss.
